client/components/Landing/LandingStats/CounterCard/CounterCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import CountUp from 'react-countup';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import styles from './CounterCard.module.scss';
+
+interface Props {
+  icon: IconProp;
+  counter: number;
+  description: string;
+  withAnimation?: boolean;
+}
+
+export default function LandingStats({
+  icon,
+  counter,
+  description,
+  withAnimation = true,
+}: Props) {
+  return (
+    <CountUp start={0} end={counter} delay={0}>
+      {({ countUpRef }) => (
+        <div className={`${styles['counter-card']} text-center`}>
+          <div
+            className={`${styles['icon-box']} d-flex justify-content-center align-items-center`}
+          >
+            <FontAwesomeIcon width="30" height="30" color="#fff" icon={icon} />
+          </div>
+          <div className={styles['counter-details']}>
+            <p ref={withAnimation && countUpRef}>{counter}</p>
+            <h3>{description}</h3>
```

Comment:
Judging from the UI, I'd say they should be inversed (the number is bigger and first).

However, I think here we should not use headings. We don't actually have a section following (like we would in an article) so maybe 2 paragraphs?